src/main/resources/application.yml

```diff
@@ -16,6 +16,10 @@ spring:
         infoBorderUrl: http://localhost:8080
     pid:
         file: application.pid
+    portal:
```

Comment:
we need about 3-5 accounts

Reply:
you can add this later, just keep note of it as a task to do. better to craete git issue if you aren't addressing in this PR

Reply:
Okay I'll add more